exercises/darts/canonical-data.json

```
@@ -1,13 +1,13 @@
 {
   "exercise": "darts",
-  "version": "2.0.0",
+  "version": "2.1.0",
   "comments": [
-    "Return the correct amount earned by a dart landing in a given point in the target problem"
+    "Return the correct amount earned by a dart landing in a given point in the target."
```

I thought maybe we could use darts-related terminology here?

```suggestion
    "Return the correct amount earned by a dart landing in a given point on the board."
```

---

Not necessarily against this (it occurred to me also), but first can you consider that its [description.md](https://github.com/exercism/problem-specifications/blob/master/exercises/darts/description.md) doesn't contain the word "board" and rather frequent use of "target".

---

Right. Just ignore my comment, I'll create a separate PR later